rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -47,6 +47,17 @@ def fromcstring3(s):
 	except AttributeError as e:
 		raise
 
+class ReturnCode:
+	ok = 0
+	no_data = 11
```

Comment:
Is there any way we can expose DDS_RETCODE_* in librtiddsconnector.so? That way we wouldn't have to maintain this list.

Reply:
We could provide some function to load the retcodes. Let's evaluate that as part of CON-77